float8_experimental/config.py

```diff
@@ -19,3 +19,8 @@
 # implements pre/post-all-gather methods to do fp8 all-gather with FSDP2.
 # Only dynamic scaling is supported for now.
 enable_fsdp_fp8_all_gather = False
+
+# If True, use 'fnuz' float8 types for calculations. If the backend
+# hardware does not support a particular dtype, the emulated implementation
```

Comment:
nit: currently the user is responsible for toggling the emulation setting, we don't do that automatically

Reply:
I thought per previous comments that we wanted to go the emulated route if the backend hardware didn't support the type? in general, the user could force to emulate, but i was under the impression that cases where a dtype was being used that wasn't supported on the underlying hardware, that we wanted to go the emulated route?

Reply:
yep, that is correct!  This is just currently done by the user explicitly, and there is no support do handle that automatically.

Reply:
oh, you're saying the comment is wrong

Reply:
fixed!